server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -365,7 +366,8 @@ private void write(final Collection<CommitSession> sessions, boolean mincFinish,
             if (commitSession.beginUpdatingLogsUsed(copy, mincFinish)) {
               try {
                 // Scribble out a tablet definition and then write to the metadata table
-                defineTablet(commitSession, writeRetry);
+                write(singletonList(commitSession), false,
+                    logger -> logger.defineTablet(commitSession), writeRetry);
```

Comment:
Is the `NO_WAIT_LOGGER_OP` behavior preserved?  Not sure if its possible at this depth in the code, but it would be more clear if define tablet were written using `Durability.LOG`.  For define tablet we want to write to the log w/o sync because we know define tablet will be followed by subsequent writes that may or may not sync.

Reply:
Yeah I see what you are saying but I am not sure how we could do that.

Reply:
Ok.  Looking at that code I think we could have a `logFileData(LogFileKey key, LogFileValue value, Durability d)` and defineTablet could call that with a durability of `LOG`.  Also there are some other method that log a single key value that could call it.   Impl could be something like :

```java
logFileData(LogFileKey key, LogFileValue value, Durability d) {
   logFileData(singletonList(new Pair(k,v)), d);
}
```

This would avoid the duplicate exception handling in defineTablet.